### PR TITLE
Added slot around menu items

### PR DESCRIPTION
**Pull request by TimDirks**

### CODE REVIEW

File: src/components/DropdownMenu/DropdownMenu.vue [hunk 1]
```diff
@@ -2,12 +2,14 @@
     <ul class="dropdown-menu">
         <slot name="prepend" />
 
-        <DropdownMenuItem
-            v-for="(item, index) of items"
-            :key="`item-${index}`"
-            v-bind="item"
-            @click="clickItem(item)"
-        />
+        <slot name="items">
```
Comment (barisbora) [inline src/components/DropdownMenu/DropdownMenu.vue:5]:
```suggestion
        <slot name="items" :click="clickItem">
```
I think we should pass the click event also so we can trigger click from custom dropdown item

**Review by barisbora — APPROVED** [review]

File: src/components/DropdownMenu/DropdownMenu.vue [hunk 2]
```diff
@@ -2,12 +2,14 @@
...
+            <DropdownMenuItem
+                v-for="(item, index) of items"
+                :key="`item-${index}`"
```
Comment (thomasowow) [inline src/components/DropdownMenu/DropdownMenu.vue:8]: We could even prevent the parent having to do the loop.
```suggestion
        <slot name="items">
            <template v-for="(item, index) of items">
                <slot name="item" :item="item">
```

File: src/components/DropdownMenu/DropdownMenu.vue (hunk 1 above)
Comment (thomasowow) [inline src/components/DropdownMenu/DropdownMenu.vue:5]:
```suggestion
        <slot name="items" :items="items">
```

**Review by thomasowow — CHANGES_REQUESTED** [review]

File: src/components/Dropdown/Dropdown.vue [hunk 3]
```diff
@@ -22,6 +22,10 @@
                 <slot name="menuPrepend" />
             </template>
 
+            <template #items>
+                <slot name="menuItems" />
+            </template>
```
Comment (thomasowow) [inline src/components/Dropdown/Dropdown.vue:27]: I think I would not be able to do the following:
```
<Dropdown>
    <template #item="{item}">
        {{ item.label }}
```
I think this also applies to `FormDropdown`. Can we have a nice eligant and flexible way to pass these slots to children?

**Review by thomasowow — APPROVED** [review]

**Review by thomasowow — APPROVED** [review]

**Pull request merged**